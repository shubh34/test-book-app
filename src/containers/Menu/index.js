import { Link } from 'react-router-dom';
import routes from '../../configs/routes';

const Menu = () => (
  <header style={{ display: 'flex', alignItems: 'center', padding: '0 16px' }}>
    <h2 style={{ padding: '16px 16px 16px 0' }}>My Books Store</h2>
    <nav>
      <Link to={routes.BOOKS.resolve()}>Home</Link>
    </nav>
  </header>
);
export default Menu;
