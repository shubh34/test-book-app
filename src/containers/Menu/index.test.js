import Menu from '.';
import { setUpConnectedComponent } from '../../../test/utils';

test('renders render Menu', () => {
  const { container } = setUpConnectedComponent({
    Component: Menu
  });
  expect(container).toMatchSnapshot();
});
