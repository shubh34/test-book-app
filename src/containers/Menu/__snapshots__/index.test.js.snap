// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders render Menu 1`] = `
<div>
  <header
    style="display: flex; align-items: center; padding: 0px 16px;"
  >
    <h2
      style="padding: 16px 16px 16px 0px;"
    >
      My Books Store
    </h2>
    <nav>
      <a
        href="/books"
      >
        Home
      </a>
    </nav>
  </header>
</div>
`;
